app/models/ext_management_system.rb

```diff
@@ -72,6 +72,7 @@ def self.api_allowed_attributes
   has_one  :iso_datastore, :foreign_key => "ems_id", :dependent => :destroy, :inverse_of => :ext_management_system
 
   belongs_to :zone
+  belongs_to :backup_zone, :class_name => "Zone", :inverse_of => :paused_ext_management_systems # used for maintenance mode
```

Comment:
This should be `:zone_before_pause` then this should be green

Reply:
@agrare Sorry, I forgot to add WIP flag, I made many rebases to reduce commits, new code is not here yet

Reply:
Ah okay :+1: